coordinator/Z-Stack_3.x.0/COMPILE.md

```diff
@@ -35,3 +35,64 @@ These changes are required to fix the coordinator crashing when the TX power is
 In SDK 7.10.02.23 TI made changes to `macSetTxPowerVal` which introduced this issue.
 The patch reverts these changes.
 The patched `maclib_*.a` files were provided by a TI employee and will only work for the 7.41.00.17 SDK.
+
+
+## Docker build environment
+
+This repo includes a Container file, to help with setting up a build environment without the need to download and install things manually. [Docker](https://docker.com) or [Podman](https://podman.io) can be used. The following example uses docker.
+
+1. This step builds the container locally. This step may be skipped if using a [released container](https://github.com/Koenkk/pkgs/container/Z-Stack-firmware) from this repository instead.
+```console
+$ docker build \
+    --file '../../Containerfile' \
+    --rm \
+    --tag 'z-stack:dev' \
+    '../../'
+```
+
+> __Note:__ The above instruction assumes we are in the coordinator/Z-Stack_3.x.0 directory, which is why we refer to the repo root. Replace `../../` with `./` if run from the root.
+
+> __:Warning:__ The build of the container will download CCS. While docker keeps a cached copy on subsequent builds, this download can take a while and is not immediately obvious it is happening.
+
+1. Enter the container so that the firmware can be built.
+```console
+$ docker run \
+    --interactive \
+    --rm \
+    --tty \
+    --volume './:/src' \
+    --volume '../../simplelink_f2_examples_sdk:/sdk' \
```

Comment:
Here, docker doens't like the volume name (https://stackoverflow.com/questions/60369990/error-installing-nextcloud-with-docker-compose ) so I used `"$PWD/../../simplelink_f2_examples_sdk:/sdk"`